tracing-core/src/field.rs

```diff
@@ -1003,25 +1017,6 @@ mod test {
         }
     }
 
-    #[test]
-    fn value_set_with_no_values_is_empty() {
-        let fields = TEST_META_1.fields();
-        let values = &[
-            (&fields.field("foo").unwrap(), None),
-            (&fields.field("bar").unwrap(), None),
-            (&fields.field("baz").unwrap(), None),
-        ];
-        let valueset = fields.value_set(values);
-        assert!(valueset.is_empty());
-    }
-
-    #[test]
-    fn empty_value_set_is_empty() {
-        let fields = TEST_META_1.fields();
-        let valueset = fields.value_set(&[]);
-        assert!(valueset.is_empty());
-    }
```

Comment:
I don't think we need these now that ValueSet can't contain "empty" values...

Reply:
Well, the individual values can't be empty, because they aren't wrapped in Option, maybe a struct with no fields could still technically have an empty ValueSet.. I need to think about this more

Reply:
Added new unit tests for `ValueSet::is_empty()`